reviewdog.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/haya14busa/reviewdog/diff"
 )
 
+const Version = "0.9.8"
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported const Version should have comment or be unexported